src/base/bittorrent/session.h

```diff
@@ -41,6 +41,7 @@
 #include <QHash>
 #include <QPointer>
 #include <QSet>
+#include <QtContainerFwd>
```

Comment:
I would sort Qt includes with different naming scheme (actually with different prefix) separately:
1. Qt-prefixed includes
2. Q-prefixed includes

(still w/o empty line between)

Reply:
I don't feel strongly, I understand that it will look good but on the other side, we have enough complexity for arranging headers. Furthermore, I wish this could be handled in another PR, this PR is not the first one to use Qt-prefixed headers and finding & changing them manually is a bit too much for me.

Reply:
Ok.